atlasdb-feign/src/main/java/com/palantir/atlasdb/http/FeignOkHttpClients.java

```diff
@@ -16,6 +16,8 @@
 package com.palantir.atlasdb.http;
 
 import java.io.IOException;
+import java.net.Proxy;
```

Comment:
Probably need to remove this - it's unused (baseline will complain)

Reply:
done